### DISCUSSION TITLE
Pybricks code group collaboration

### DISCUSSION THREAD

**Discussion by Barry-Williams** [pybricks/support#811] [Ideas]
I am a high school teacher we currently use EV3s and with both stock firmware and pybricks.
We have just added 2 spike prime sets to our collection both to replace the ailing ev3s but also take advantage of the simplicity of using a web code editor instead of VS Code (which our IT department will not let us install on school devices).
Our students work in groups of 2 or 3 and I would like a simple way to have any student access this code (in case one is away). This could be achieved a few ways but I would like the simplest option requiring the minimal steps from students. 
E.G. While it is possible with the current pybricks website (or beta) to export the code download it to a shared onedrive folder. These steps are likely to be forgotten. 
With Pybricks on the EV3 students could recover the project back from the EV3 through SSH or exploring the file system in VS Code. I am not sure if this is possible with pybricks. 
An alternative is to add cloud storage of some kind that will save the project to a shared folder automatically. 
Github would also be a possibility but only one student will be coding at a time so they don't need the power of source control (and setting up a GitHub account, commits, etc. is again more steps than I would like. 
One other thing that would work is coding on iPads (while not ideal for other reasons) We do have a set of iPads that live in the classroom that were used for block coding. This means that the code is stored on the iPad and is also always available. Unfortunately, there are issues with iOS and BLE. I have seen the discussions about alternative browsers but I am not sure how good that solution is.

**Comment by laurensvalk** [comment]
Thanks for opening this issue. This would be very nice indeed but unfortunately it's not something we have in the pipeline for now.
